concepts/ast/about.md

```diff
@@ -0,0 +1,40 @@
+# About
```

Comment:
I gave up on writing a custom `about.md` and just copy-pasted `introduction.md` because I have doubts about how many students actually read those and I'm low on time.

Reply:
I think we will need to circle back to it once we have quote/unquote and macros concepts anyway.